cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetricsTable.java

```diff
@@ -38,7 +38,7 @@
   /**
    * Write multiple rows, each with multiple individual columns to write.
    */
-  void put(SortedMap<byte[], ? extends SortedMap<byte[], Long>> updates);
+  void put(SortedMap<byte[], ? extends SortedMap<byte[], byte[]>> updates);
```

Comment:
how come this is being changed to byte[]? It *must* be a Long, otherwise everything breaks. Even if everybody passes in a byte representation of a long, it is bad that the api does not enforce the type.